### ISSUE TITLE
[bug]: TypeError: i.map is not a function

### ISSUE THREAD

**Issue by jackywackywoo** [invoke-ai/InvokeAI#6936]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### Operating system

Windows

### GPU vendor

None (CPU)

### Version number

5.0.0

### Browser

Firefox 130.0.1

### What happened

Attempted to load Invoke via the .bat file, selecting option one. Once it had loaded, I went to the web address and was immidiately met with the error. 

### What you expected to happen

I expected the UI to load to allow me to begin.

### How to reproduce the problem

Load Invoke as normal

### Additional context

pressing the reload UI button reloads the page, but the same error persists.

**Comment by psychedelicious** [comment]
Hi, this has popped up before but its very mysterious. I haven't been able to reproduce it ever.

Can you please try in a private browsing window to see if that fixes the issue?

If that doesn't work, please try FF's Troubleshoot Mode, which disables all extensions: https://support.mozilla.org/en-US/kb/diagnose-firefox-issues-using-troubleshoot-mode

I wonder if a setting or extension isn't playing well with the UI.

**Comment by jackywackywoo** [comment]
Huh, trying to connect in a private window worked! No issues at all. 

The only extension I have enabled is Ublock Origin, which is also active in a private window.

Thank you for your help! Almost annoyed I didn't think to try that myself...

**Comment by psychedelicious** [comment]
@jackywackywoo That's good to hear, but its not acceptable that the app only works in a private window. Should work normally too of course. 

Back in a non-private window, can you try clearing cache and cookies? You can do it only for Invoke with this button:
<img width="455" alt="image" src="https://github.com/user-attachments/assets/84423afb-58b6-4031-8812-9c7651b0e6aa">

**Comment by jackywackywoo** [comment]
I agree with you there, so you may be pleased to hear that clearing the cookies for Invoke has in fact gotten it working. (which only makes me feel more like an idiot)

I had upgraded from a much older version of Invoke, so perhaps there was something leftover that was causing a conflict?

Either way, thank you again!

**Comment by psychedelicious** [comment]
> I had upgraded from a much older version of Invoke, so perhaps there was something leftover that was causing a conflict?

Ahh, that's gotta be it. Theoretically, the app should have reset all that old data when you clicked the reset button. I'll review that button and see if maybe it's missing some stuff.

**Comment by NemukaPrime** [comment]
I just got this error. 
Because of using other WebUIs, I had pinned tabs in Firefox with the usual URL: http://127.0.0.1:7860/
But I loaded InvokeAI in one with a dark mode part at the end, after just changing the port to 9090 as mentioned in the terminal window, like so: http://127.0.0.1:9090/?__theme=dark

I got the same error afterward.

> TypeError: i.map is not a function

I tried things like allowing extracting canvas data, but none of the usual things solved it. But when I went to http://127.0.0.1:7860/, it finally worked.

On my end, it has been fixed. Just letting you know of my solution.
